docs/source/reference/frameworks/fastai.rst

```diff
@@ -0,0 +1,27 @@
+======
+FastAI
+======
+
+.. admonition:: About this page
+
+   This is an API reference for FastAI in BentoML. Please refer to
+   :ref:`../.../framework/fastai` for more information about how to use FastAI
+   in BentoML.
+
+
+.. currentmodule:: bentoml.fastai
+
+.. autofunction:: bentoml.fastai.save_model
+
+.. autofunction:: bentoml.fastai.load_model
+
+.. autofunction:: bentoml.fastai.get
+
+
+.. admonition:: Help us improve the project!
```

Comment:
remove this? we probably don't need this on every page

Reply:
noted.